client/src/app/shared/components/composable-app-table/composable-app-table.tsx

```diff
@@ -0,0 +1,215 @@
+          {toolbarActions}
+          {!withoutTopPagination && (
+            <ToolbarItem
+              variant={ToolbarItemVariant.pagination}
+              alignment={{ default: "alignRight" }}
+            >
+              <SimplePagination
+                idPrefix={paginationIdPrefix}
+                isTop={true}
+                paginationProps={paginationProps}
+              />
+            </ToolbarItem>
+          )}
+        </ToolbarContent>
+      </Toolbar>
+      <TableComposable aria-label="waves-table" variant={variant}>
+        <Thead>
+          <Tr>
+            {Object.keys(columnNames).map((name: string) => (
+              <Th>{columnNames[name]}</Th>
+            ))}
+            <Th />
+          </Tr>
+        </Thead>
+        {isLoading && (
+          <Tbody>
+            <Tr>
+              <Td colSpan={8}>
+                <Bullseye>
+                  <Spinner size="xl" />
+                </Bullseye>
+              </Td>
+            </Tr>
+          </Tbody>
+        )}
+        {fetchError && (
+          <Tbody aria-label="Table error">
+            <Tr>
+              <Td colSpan={8}>
+                <Bullseye>{errorState ? errorState : <StateError />}</Bullseye>
+              </Td>
+            </Tr>
+          </Tbody>
+        )}
+
+        {rowItems.map((row: IComposableRow, rowIndex: number) => {
+          return (
+            <Tbody key={row.id} isExpanded={row.isExpanded}>
+              <Tr>
+                {isSelectable &&
+                  handleToggleRowSelected &&
+                  handleIsRowSelected && (
+                    <Td
+                      select={{
+                        rowIndex,
+                        onSelect: (_event, isSelecting) => {
+                          handleToggleRowSelected(row, isSelecting);
+                        },
+                        isSelected: handleIsRowSelected(row),
+                      }}
+                    />
+                  )}
+                {row.cells.map((cell: IRowCell, i) => (
+                  <Td
+                    width={cell.width}
+                    dataLabel={cell.id}
+                    component="th"
+                    compoundExpand={cell.compoundExpand}
+                    className={cell.className}
+                    style={cell.style}
+                  >
+                    {cell.title}
+                    {cell.children}
+                  </Td>
+                ))}
+              </Tr>
+              {!!row.expandedContentMap && row.isExpanded ? (
+                <Tr isExpanded={row.isExpanded}>
+                  <Td
+                    dataLabel={columnNames[row.expandedCellKey]}
+                    noPadding
+                    colSpan={8}
+                    width={100}
+                  >
+                    {Object.keys(row.expandedContentMap).map(
+                      (expandableRowName: string) => {
+                        if (row.expandedCellKey === expandableRowName) {
```

Comment:
If the expandable row is clicked && there is expandable content available passed in via the expandedContentMap prop for the table row, https://github.com/konveyor/tackle2-ui/pull/598/files#diff-f327e9221e5058c233a30a6c593caec4943e2e94853c7abf11fc4cb01801d6b7R39 

then render the mapped content that has been passed in 

![image](https://user-images.githubusercontent.com/11218376/224821320-fe9d6e21-2448-47ac-b07f-2967787d3a32.png)